spec/javascript/components/lease_ups/LeaseUpApplicationsPage.test.js

```diff
@@ -203,8 +203,8 @@ describe('LeaseUpApplicationsPage (shallow)', () => {
           expect(mockFetchLeaseUpApplications.mock.calls[1][0]).toEqual(mockListing.id)
 
           // It's confusing but this page param is actually the server page, so even though
-          // the client, "eager" page is 1 the server page is still 0.
-          expect(mockFetchLeaseUpApplications.mock.calls[1][1]).toEqual(0)
+          // the client, "eager" page is 6 the server page is only 1.
```

Comment:
I thought page 6 should be page 2 on the server? I thought we switched to  new server page every 100 records/5 pages

Reply:
@akegan yeah the page is zero indexed so this is page 2